code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -286,3 +286,28 @@
 	name = "syringe (syriniver)"
 	desc = "Contains syriniver, used to treat toxins and purge chemicals.The tag on the syringe states 'Inject one time per minute'"
 	list_reagents = list(/datum/reagent/medicine/C2/syriniver = 15)
+
+/obj/item/reagent_containers/syringe/contraband

```

Comment:
Would it be possible to make subtypes of this contain a random volume of chems, with some extra weighting for "0", for that real "spent prison syringe" feel?

Also, consider adding a chance for these to come with labels that may or may not be accurate (70% chance to be accurate, perhaps?).

Reply:
I kinda like the idea, although I'm not sure I'll get around to it on this first PR.

Reply:
Do it yourself.